examples/ur5e/get_joint_info.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np
+
+import airobot as ar
+
+
+def main():
```

Comment:
@taochenshh please include doc-strings of the what the main fucntion does.

Reply:
commit 5ae7bdbb7995390de89fb7a93095fab0e41abe50